pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1073,8 +1083,6 @@ cdef class _Timedelta(timedelta):
         >>> td.days
         0
         """
-        # NB: using the python C-API PyDateTime_DELTA_GET_DAYS will fail
-        #  (or be incorrect)
```

Comment:
if these comments are no longer accurate, can/should we use the referenced APIs?